### PR TITLE
update import in doc.go of staging/src/k8s.io/metrics

### PR DESCRIPTION
**Pull request by bobsongplus**
**What this PR does / why we need it**:
   update import in doc.go of staging/src/k8s.io/metrics
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Only a partial fix for #68231.

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by TinySong** [comment]
Can I add a comment for doc.go  in this PR ?

**Comment by DirectXMan12** [comment]
this needs an actual description of why this is being done, for posterity's sake.

File: staging/src/k8s.io/metrics/pkg/apis/external_metrics/doc.go [hunk 1]
```diff
@@ -17,4 +17,4 @@ limitations under the License.
 // +k8s:deepcopy-gen=package
 // +groupName=external.metrics.k8s.io
 
-package external_metrics
+package external_metrics // import "k8s.io/metrics/pkg/apis/external_metrics"
```
Comment (nikhita) [inline staging/src/k8s.io/metrics/pkg/apis/external_metrics/doc.go]: There needs to be a newline after this line

**Comment by nikhita** [comment]
@TinySong can you update the commit message and PR body to say something like this?

```
Add canonical import paths to metrics packages

The comment will ensure that only imports with
this path will compile.This is needed to make sure
that the vanity url k8s.io is used as the import path
instead of github.com.

See https://golang.org/doc/go1.4#canonicalimports
for more details.
```

**Comment by idealhack** [comment]
/retest

cc @kubernetes/sig-instrumentation-pr-reviews

**Pull request closed without merging**